### PR TITLE
Bump Substrate to `b094edaf`

### PR DESCRIPTION
**Pull request by HCastano**

### CODE REVIEW

File: bin/millau/node/src/cli.rs [hunk 1]
```diff
@@ -63,7 +63,7 @@ pub enum Subcommand {
 	Revert(sc_cli::RevertCmd),
 
 	/// Inspect blocks or extrinsics.
-	Inspect(node_inspect::cli::InspectCmd),
+	Inspect(node_inspect::cli::InspectKeyCmd),
```
Comment (tomusdrw) [inline bin/millau/node/src/cli.rs:66]: :thinking:

Comment (HCastano) [reply bin/millau/node/src/cli.rs:66]: @tomusdrw What's with the 🤔?

Comment (tomusdrw) [reply bin/millau/node/src/cli.rs:66]: I guess this search & replace didn't go that well. 
https://github.com/paritytech/substrate/pull/8678/commits/3700a10625767ee9a76731fcb57b5638a5c33081

`node_inspect` has nothing to do with keys and should not have been touched in th original PR. CC @NukeManDan

Comment (NukeManDan) [reply bin/millau/node/src/cli.rs:66]: My mistake. I should have caught this, likely why the struct naming was called out in that PR. All good now?

Comment (NukeManDan) [reply bin/millau/node/src/cli.rs:66]: Ah or is a new PR to fix this issue needed in both places?

Comment (HCastano) [reply bin/millau/node/src/cli.rs:66]: @NukeManDan So we'll need a Substrate PR to revert the changes, and then a PR which bumps Substrate in this repo (and in that PR changes the name back)

Comment (NukeManDan) [reply bin/millau/node/src/cli.rs:66]: https://github.com/paritytech/substrate/pull/8787
Once this lands, I will open a PR to bump substrate and the name change.

Comment (tomusdrw) [reply bin/millau/node/src/cli.rs:66]: Awesome, thanks! We will fix that during our regular substrate-bump cycle, so you don't have to do it unless you really want to :) If you run into any other issues (that might be related to substrate bump - let us know).

**Pull request merged**